### PR TITLE
Update Windows resource files (and add one for bitcoin-tx)

### PR DESCRIPTION
**Pull request by luke-jr**
I have not tested this at all.

### CODE REVIEW

**Comment by jgarzik** [comment]
ut ACK

**Comment by laanwj** [comment]
@theuni This changes the build system to add a windows resource file, can you take a look if it's correct?

**Comment by theuni** [comment]
Looks good to me. Travis builds and inserts these, so we can be somewhat confident that they're working as intended. utACK.

File: src/bitcoind-res.rc [hunk 1]
```diff
@@ -17,13 +17,13 @@ BEGIN
         BLOCK "040904E4" // U.S. English - multilingual (hex)
         BEGIN
             VALUE "CompanyName",        "Bitcoin"
-            VALUE "FileDescription",    "Bitcoind (OSS daemon/client for Bitcoin)"
+            VALUE "FileDescription",    "bitcoind (Bitcoin client as a JSON-RPC server)"
```
Comment (sipa) [inline src/bitcoind-res.rc]: JSON-RPC is hardly the only thing it does. I would just say headless version or version without GUI.

Comment (luke-jr) [inline src/bitcoind-res.rc]: It's the only user interface... "headless" or "without GUI" implies it is suitable for human use.

Comment (sipa) [inline src/bitcoind-res.rc]: But a Bitcoin node is useful even if you don't use its RPC functionality,
for example by connecting a lightweight client using P2P to it.

Bitcoind is bitcoin-qt without the GUI part, so it's a headless version -
regardless of how you use it.

Comment (jgarzik) [inline src/bitcoind-res.rc]: +1 @sipa

File: src/bitcoin-cli-res.rc [hunk 2]
```diff
@@ -17,13 +17,13 @@ BEGIN
         BLOCK "040904E4" // U.S. English - multilingual (hex)
         BEGIN
             VALUE "CompanyName",        "Bitcoin"
-            VALUE "FileDescription",    "Bitcoin-cli (OSS RPC client for Bitcoin)"
+            VALUE "FileDescription",    "bitcoin-cli (JSON-RPC client for Bitcoin Core)"
```
Comment (Diapolo) [inline src/bitcoin-cli-res.rc]: So it is not Open source anymore?

Comment (laanwj) [inline src/bitcoin-cli-res.rc]: I'm not sure license information belongs in the simple file description (and OSS is not a very common abbreviation in any case)

File: src/bitcoind-res.rc (hunk 1 above)
Comment (laanwj) [inline src/bitcoind-res.rc]: I like "headless Bitcoin node". "Client" is also the wrong word to use in a P2P world.

Comment (luke-jr) [inline src/bitcoind-res.rc]: So... "bitcoind (Bitcoin node with a JSON-RPC server)"?

Comment (laanwj) [inline src/bitcoind-res.rc]: Fine with me.

**Comment by laanwj** [comment]
@luke-jr are you planning to update the name so that this can be merged?

**Comment by luke-jr** [comment]
@laanwj Woops, thought I already had,  sorry. Done.

File: src/qt/res/bitcoin-qt-res.rc [hunk 3]
```diff
@@ -19,7 +19,7 @@ BEGIN
         BLOCK "040904E4" // U.S. English - multilingual (hex)
         BEGIN
             VALUE "CompanyName",        "Bitcoin"
-            VALUE "FileDescription",    "Bitcoin Core (OSS GUI client for Bitcoin)"
+            VALUE "FileDescription",    "Bitcoin Core (GUI client for Bitcoin)"
```
Comment (laanwj) [inline src/qt/res/bitcoin-qt-res.rc]: Ugh, we have to client->node here to, to be consistent with the bitcoind change ("GUI Bitcoin node"?)

Comment (luke-jr) [inline src/qt/res/bitcoin-qt-res.rc]: Fixed

**Pull request merged**